__tests__/app.test.js

```diff
@@ -148,4 +149,51 @@ describe('/api/articles' , () => {
             })
         })
     })
+})
+
+describe('/api/articles/:article_id/comments' , () => {
+    describe('GET' , () => {
+        test('GET: 200 - returns with an array of comment objects relating to the specified article_id' , () => {
+            return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({body: {comments}}) => {
+                expect(comments.length).toBe(11)
+                comments.forEach((comment) => {
+                    expect(comment).toEqual({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        article_id: 1
+                    })
+                })
+            })
+        })
+        test('GET: 200 - the returned array sorts the comments by date (descending' , () => {
+            return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({body: {comments}}) => {
+                expect(comments).toBeSortedBy('created_at' , {descending: true})
+            })
+        })
```

Comment:
We should add a test to check for articles that have no comments attached to them, I believe article number 2 would be one of them. The response should be 200 with an empty array :)